src/azul/plugins/metadata/hca/transform.py

```diff
@@ -1132,15 +1132,13 @@ def matrix_stratification_values(self, file: api.File) -> JSON:
                     contents['donors'].append(donors)
                 organ = stratum.get('organ')
                 if organ is not None:
-                    organ = sorted(organ)
-                    contents['samples'].append(
-                        {
-                            'biomaterial_id': f'specimen_from_organism_{file_name}',
-                            'entity_type': 'specimens',
-                            'organ': organ,
-                            'effective_organ': organ,
-                        }
-                    )
+                    for i, one_organ in enumerate(sorted(organ)):
+                        contents['specimens'].append(
+                            {
+                                'biomaterial_id': f'specimen_from_organism_{i}_{file_name}',
```

Comment:
```suggestion
                                'biomaterial_id': f'specimen_from_organism_{i}_{file_name}',
```

How come this change isn't reflected in any of the tests? Should I at least see `_0` in some of the assertions?